llvm/lib/IR/RuntimeLibcalls.cpp

```diff
@@ -65,7 +65,17 @@ static void setAArch64LibcallNames(RuntimeLibcallsInfo &Info,
 #undef LCALLNAME5
 }
 
-static void setARMLibcallNames(RuntimeLibcallsInfo &Info, const Triple &TT) {
+static void setARMLibcallNames(RuntimeLibcallsInfo &Info, const Triple &TT,
+                               FloatABI::ABIType FloatABIType,
+                               EABI EABIVersion) {
```

Comment:
The EABIVersion doesn't seem to be used by this function, or at least not yet.

Looks like it is only used in https://github.com/llvm/llvm-project/blob/main/llvm/lib/Target/ARM/ARMISelLowering.cpp#L693

Is the intention to move that part of the code to here at some point?

My apologies if I've missed some context, just seeing these reviews for the first time.